src/services/useActivity.ts

```diff
@@ -64,6 +64,27 @@ export const useActivityRequests = () => {
     [axiosLoggedRequest, enqueueSnackbar, queryClient],
   );
 
+  const askSameQuestion = React.useCallback(
+    async (activity: Activity, data: AnyData) => {
+      const response = await axiosLoggedRequest({
+        method: 'PUT',
+        url: `/activities/${activity.id}/askSame`,
+        data: {
+          data,
+        },
+      });
```

Comment:
There's no need to send data here, the backend API you created don't use it. So just do:
```tsx
const response = await axiosLoggedRequest({
  method: 'PUT',
  url: `/activities/${activity.id}/askSame`,
});
```